app/__init__.py

```diff
@@ -67,5 +68,17 @@
     # TODO - No need of this for REST
     app.logger.info('ProgDay startup')
 
+# defining the errors blueprint
+from app.errors import bp as errors_bp
+app.register_blueprint(errors_bp)
+
```

Comment:
from app import errors, auth
app.register_blueprint(errors.bp)
app.register_blueprint(auth.bp)
Could be better?